qa/TL3_RN50_short/test_pytorch.sh

```diff
@@ -31,9 +31,9 @@ if [[ $RET -ne 0 ]]; then
     CLEAN_AND_EXIT 2
 fi
 
-MIN_TOP1=10.0
-MIN_TOP5=20.0
-MIN_PERF=50
+MIN_TOP1=20.0
+MIN_TOP5=40.0
+MIN_PERF=400
```

Comment:
Maybe with the local data we can bump up the threshold, something to consider.

Reply:
Sure, I will look into that, I wanted to have a test run first to learn more.